Model/ContentRepository.php

```diff
@@ -344,20 +344,23 @@ public function findSortedByTemplateTitle($template, $order = 'ASC')
      */
     public function findRelated(Content $content, $limit = 10)
     {
+        $city = $content->getValueSet()->getValueFor('address')->getAddress()->getCity();
```

Comment:
I don't think this belongs here... Maybe we should mark this method as deprecated as well. What do you think?